quantum/rgb_matrix_animations/rgb_matrix_effects.inc

```diff
@@ -32,3 +32,6 @@
 #include "rgb_matrix_animations/solid_reactive_nexus.h"
 #include "rgb_matrix_animations/splash_anim.h"
 #include "rgb_matrix_animations/solid_splash_anim.h"
+#include "rgb_matrix_animations/hue_breathing_anim.h"
```

Comment:
Actually, could you move these to right after jellybean_raindrops (and before typing heatmap?

Reply:
Done! Thank you